cas-server-core/src/main/java/org/jasig/cas/CasVersion.java

```diff
@@ -43,4 +47,27 @@ private CasVersion() {
     public static String getVersion() {
         return CasVersion.class.getPackage().getImplementationVersion();
     }
+
+    /**
+     * Gets last modified date/time for the module.
+     * @return the date/time
+     */
+    public static DateTime getDateTime() {
```

Comment:
Most if not all CAS themes completely rip out the login screen; by that logic the CASVersion class is totally useless. Maybe we could log this when the context starts up as well?